### DISCUSSION TITLE
Disallow legendary items in arena.

### DISCUSSION THREAD

**Discussion by TotallyNotMyAccount** [chromiecraft/chromiecraft#7053] [Suggestions]
While it is somewhat funky on CC because we are at level 70, wrath pvp is regarded by the wider community as one of the best pvp metas this game ever had. The only pain point that regularly pops up are the legendaries, that warp balance in a very bad way. With the advent of T6 we are getting close to the first pair of these problematic items becoming available. My hope is that we nip this in the bud as soon as possible, while also future proofing against what is to come with Val'anyr and especially Shadowmourne.

**Comment by heyitsbench** [comment]
This'll be fun.

**Comment by tbcxd** [comment]
Hard disagree.

**Reply by TotallyNotMyAccount** [reply]
Reasons? Any kind of argument beyond: "I want 150 more rating than my skill would normally get me."?

**Reply by tbcxd** [reply]
If we remove sm, we could also suggest removing heroic version on trinkets in pvp because they make casters disgustingly strong and then *this item* and that item and so on, leading to constant suggestions of removing items.

I for one, do not agree with humans being able to use double solance, normal and heroic version because it gives casters too much mp5, let's remove it *shrug*

**Reply by tbcxd** [reply]
Don't take this as a personal attack, because you know I respect you, but it seems quite hypocritic saying "I want 150 more rating than my skill allow me" coming from someone that plays 3s with enh and arms warrior.

**Reply by TotallyNotMyAccount** [reply]
Being able to find good partners and picking a good comp is part of arena skill. And yes, pve trinkets and other gear are also problematic. But realistically there's no way we ever get a ban on all pve items in arena, so I won't bother advocating for it. Same with humans being OP, CC staff will never touch it, so why bother raising a fuss over it. Legendaries might actually happen and a ban would be hugely beneficial for the meta and the little guy. But sure, leave them in. I guess I'll have fun rolling people with val'anyr + 2 Smourn TSG.
